client-base/src/main/kotlin/app/cash/backfila/client/spi/BackfilaClientServiceHandler.kt

```diff
@@ -82,6 +83,16 @@ class BackfilaClientServiceHandler @Inject constructor(
     }
   }
 
+  @Throws(UnknownBackfillException::class)
+  fun finalizeBackfill(request: FinalizeBackfillRequest) {
```

Comment:
We need to pass in the request and make sure that it includes parameters, you may need them here too. Also we should create a response class.